app/models/manager_refresh/inventory_object.rb

```diff
@@ -69,6 +69,10 @@ def attributes(inventory_collection_scope = nil)
       attributes_for_saving
     end
 
+    def assign_attributes(attributes)
+      attributes.each { |k, v| @data[k] = v if respond_to?("#{k}=") }
```

Comment:
if we would want to use the writers, we probably want the exception thrown, when using not existing, so i would delete the if respond_to?("#{k}=")